src/app/components/sites/pages/edit/edit.component.ts

```diff
@@ -84,6 +84,7 @@ SiteEditComponent.linkToRoute({
     [projectKey]: projectResolvers.show,
     [siteKey]: siteResolvers.show,
   },
+  title: () => "Edit",
```

Comment:
Seems like we'd probably use the title "edit" a few times. Refactor common titles into some constants?